package.json

```diff
@@ -15,6 +15,7 @@
     "axios": "^0.19.2",
     "expo": "~36.0.2",
     "react": "~16.13.0",
+    "react-bottom-scroll-listener": "^3.0.0",
```

Comment:
I think you are not using it, are you? 😛

Reply:
I found it and tried to use it here but didn't work. Is it just for plain react web apps?

Reply:
Yep, just React from what I see. I added this comment as if you're not using it then there's no point in keeping it in `package.json` -> please remove it and run `npm install` again :)